endpoints/cookie_sync_test.go

```diff
@@ -2207,3 +2274,145 @@ func getDefaultActivityConfig(componentName string, allow bool) *config.AccountP
 		},
 	}
 }
+
+func TestSetCookieDeprecationHeader(t *testing.T) {
+	getTestRequest := func(addCookie bool) *http.Request {
+		r := httptest.NewRequest("POST", "/cookie_sync", nil)
+		if addCookie {
+			r.AddCookie(&http.Cookie{Name: receiveCookieDeprecation, Value: "1"})
+		}
+		return r
+	}
+
+	type args struct {
+		w       http.ResponseWriter
+		r       *http.Request
+		account *config.Account
```

Comment:
Nitpick: Please use more description names for `w` and `r`. You may wish to include the args directly in the tests struct to simplify the test a bit. Something like:

```
tests := []struct {
		name                            string
		responseWriter                  http.ResponseWriter
		request                         *http.Request
		account                         *config.Account
		expectedCookieDeprecationHeader bool
	}{
```

Reply:
done